Xamarin.PropertyEditing.Mac/PropertyList.cs

```diff
@@ -13,6 +13,9 @@ internal class PropertyList
 	{
 		internal const string PropertyEditorColId = "PropertyEditors";
 
+		public override bool AcceptsFirstResponder () => true;
+		public override bool BecomeFirstResponder () => Window.MakeFirstResponder (propertyTable);
```

Comment:
```suggestion
		public override bool BecomeFirstResponder () => Window?.MakeFirstResponder (propertyTable);
```

Reply:
This will throw a compile error due to the possibility of a Window being null, which isn't compatible with the return type of bool. We'd need a bool?

Reply:
Window?.MakeFirstResponder (propertyTable) ?? false